diff --git a/node_modules/@sveltejs/kit/dist/ssr.js b/node_modules/@sveltejs/kit/dist/ssr.js
index 32b08df..858ad88 100644
--- a/node_modules/@sveltejs/kit/dist/ssr.js
+++ b/node_modules/@sveltejs/kit/dist/ssr.js
@@ -1462,11 +1462,6 @@ function get_multipart(text, boundary) {
 			if (name === 'content-disposition') {
 				if (value !== 'form-data') nope();
 
-				if (directives.filename) {
-					// TODO we probably don't want to do this automatically
-					throw new Error('File upload is not yet implemented');
-				}
-
 				if (directives.name) {
 					key = directives.name;
 				}
